samples/EventHub/allowed-event-hub-firewall-ip/azurepolicy.json

```diff
@@ -0,0 +1,39 @@
+{
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "name": "allowed-event-hub-firewall",
+  "properties": {
+    "displayName": "Allow IP for event hub firewall",
```

Comment:
>Allow IP for event hub firewall [](start = 20, length = 31)

Can you update this to "Event Hub firewall should only allow certain IPs" to match naming standards?

Reply:
Done